source/terms.txt

```diff
@@ -0,0 +1,169 @@
+       | Only prepend the function name with the package that it comes from when disambiguation seems necessary or when listing links to API documentation.
+       | Examples:
+
+       - "Call the ``NewColor()`` function"
+       - "You can call the ``color.Rename()`` or ``person.Rename()`` method."
+
+   * - **method**
+     - | **Description:**
+       | A function declared with a receiver.
+       | **Recommendation:**
+       | Refer to this as a "method" if you require the non-specific form. Otherwise, use the name within monospace with ``()`` appended to the end of the name, and omit parameter information unless describing the signature. Ensure that you include context on what the receiver is.
+       | Only prepend the function name with the package that it comes from when disambiguation seems necessary or when listing links to API documentation.
+       | Example:
+       | Assume the documentation refers to this method:
+
+       .. code-block:: go
+
+          func (a *Color) isPrimary() bool {}
+
+       | "You can call the ``isPrimary()`` method on a ``Color`` pointer receiver type to determine whether the color is red, yellow, or blue as shown below:"
+
+       .. code-block:: go
+
+          // assume assignment c := NewColor("red")
+          c.isPrimary()
+
+   * - **return values/lists**
+     - | **Description:**
+       | The data returned from a method or function. You can find the type and name (if named) parameters in the method/function signature in what we call the **result** or **result list**. See **parameter**.
+       | **Recommendation:**
+       | Refer to these as return values or return lists. See the **parameter** term for an example of its usage.
+
+   * - **argument**
+     - | **Description:**
+       | Refers to non-specific data that you pass to a method or function.
+       | **Recommendation:**
+       | Use this term when you need to describe the data you're passing without specific details like type, usually to refer to position in the method/function signature.
+
+       | Here are a few examples of how to use this term:
+       - "Pass your connection string as the second argument to ``mongo.Connect()``"
+       - "The ``NewColor()`` function takes a ``string`` argument."
+       - `Golang.org reference docs <https://golang.org/ref/spec#Passing_arguments_to_..._parameters>`__
+
+   * - **value** (passed)
+     - | **Description:**
+       | Similar to the `wikipedia definition of value <https://en.wikipedia.org/wiki/Value_(computer_science)>`__ in that it is the "value of a variable" that can be of any data type. This is different from **return values/lists** in that it refers to the contents of the type. See `Go by Example: Values <https://gobyexample.com/values>`__ for examples.
+       | **Recommendation:**
+       | **TODO:** decide whether this term should be formatted in monospace, quotes, or both.
```

Comment:
Good question. Are there any other terms (not in-code examples of terms) that receive special formatting? 

I can't think of any of the top of my head, so my gut is that the term value should not receive special formatting. I am open to an argument that it should however.